IM150/admin/admin_forum_tour.php

```diff
@@ -163,7 +165,7 @@ function reorder_pages()
 			$message = prepare_message(trim($message), 1, 1, 1, $bbcode_uid);
 
 			$sql = "UPDATE " . FORUM_TOUR_TABLE . "
-				SET page_subject = '$subject', page_text = '$message', page_access = $page_access
+				SET page_subject = '$subject', page_text = '$message', page_access = '$page_access'

```

Comment:
Subject and body are strings, yes, not sure about access. I'll check a bit later...